src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -79,6 +79,7 @@ $spacing: 20px;
 
   &__events {
     max-height: 35vh;
+    overflow-x: hidden;
```

Comment:
This could make it an issue of clipping the content that was making the horizontal scrollbar show up. 

Is the text (or whichever items are displayed in it) doing its re-flow automatically or is it clipped?

Reply:
They get re-flowed as in the below screenshot. I would track down exactly what is causing the horizontal scroll to appear, but, we want to do the bare minimum with AV1 for the upcoming parties and then get rid of it.

![image](https://user-images.githubusercontent.com/88904/143469446-9795338c-0abb-4d87-a88c-063bc31b0080.png)

Reply:
Sounds reasonable.